android/coretest/src/androidTest/java/com/mongodb/stitch/android/core/StitchAppClientIntTests.kt

```diff
@@ -304,4 +304,56 @@ class StitchAppClientIntTests : BaseStitchAndroidIntTest() {
             )
         }
     }
+
+    @Test
+    fun testCallResetPasswordFunction() {
+        val app = createApp()
+
+        val function = app.second.functions.create(FunctionCreator(
+            "testResetPasswordFunction",
+            """
+            exports = function({email, password}, arg1, arg2) {
+                if (arg1 == 0 && arg2 == 1) {
+                    return { "status": "success" };
+                } else {
+                    return { "status": "fail" };
+                }
```

Comment:
might want to add a case for pending and just make sure that the password is unchanged